topostats/validation.py

```diff
@@ -73,24 +73,52 @@ def validate_config(config: dict, schema: Schema, config_type: str) -> None:
             ),
             "otsu_threshold_multiplier": float,
             "threshold_std_dev": {
-                "below": lambda n: n > 0,
-                "above": lambda n: n > 0,
+                "below": [
```

Comment:
We want to avoid having negative numbers passed  and I've not tried this but could you perhaps put the lambda within the `Or()`?

```
Or(
    None,
    lambda n: n > 0,
    error=("Whoops!")
)
```

Reply:
Sadly that causes this error for all of them:
```
Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniforge/base/envs/new/lib/python3.10/site-packages/schema.py", line 405, in validate
    nvalue = Schema(svalue, error=e, ignore_extra_keys=i).validate(value, **kwargs)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/new/lib/python3.10/site-packages/schema.py", line 409, in validate
    raise SchemaError([message] + x.autos, [e.format(data) if e else None] + x.errors)
schema.SchemaError: Key 'threshold_std_dev' error:
Key 'below' error:
<lambda>([10.0, None]) raised TypeError("'>' not supported between instances of 'list' and 'int'")
```